README.md

```diff
@@ -7,11 +7,11 @@ This project provides an isomorphic javascript package for accessing Azure. Righ
 
 - nodejs
 ```
-npm install azure-batch-js
+npm install azure-batch-ts
```

Comment:
Oh I remember now. I intentionally changed it to -js (since we wanted to move ahead with the plan of azure-batch-js) to indicate browser javascript (since TS is only a compile time thing). I think you made the changes in your source code to consume -js accordingly. Can you please regenerate again with azure-batch-js as the package name?